playbooks/templates/dev-install_net_config.yaml.j2

```diff
@@ -26,6 +26,14 @@ network_config:
   - type: interface
     name: dummy0
     nm_controlled: true
+{% for ip in tunnel_remote_ips | default([]) %}
```

Comment:
nit: the default here is redundant, as we already default it in vars/defaults.yaml

Reply:
ack